universe/ValueRef.cpp

```diff
@@ -632,8 +632,8 @@ double Variable<double>::Eval(const ScriptingContext& context) const
     IF_CURRENT_VALUE(float)
 
     if (m_ref_type == NON_OBJECT_REFERENCE) {
-        if (property_name == "UniverseCentreX" |
-            property_name == "UniverseCentreY")
+        if ((property_name == "UniverseCentreX") |
```

Comment:
The change is just to remove the spurious warning, the compiler warning `-Wparentheses` wants the `&&` clause explicitly bracketed.